superchain/configs/mainnet/soneium.toml

```
@@ -0,0 +1,62 @@
+name = "Soneium"
+chain_id = 1868
+public_rpc = "https://rpc.soneium.org"
+sequencer_rpc = "https://rpc.soneium.org"
+explorer = "https://soneium.blockscout.com/"
+superchain_level = 0
+governed_by_optimism = true
+standard_chain_candidate = true # This is a temporary field which causes most of the standard validation checks to run on this chain
+batch_inbox_addr = "0x008dC74CecC9dedA8595B2Fe210cE5979F0BfA8e"
+canyon_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+delta_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+ecotone_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+fjord_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+granite_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+holocene_time = 1738573200 # Mon 3 Feb 2025 09:00:00 UTC
```

```suggestion
holocene_time = 1738573200 # Mon 3 Feb 2025 09:00:00 UTC
superchain_time = 1738573200 # Joining from Isthmus
```
If possible, I'd be great to also join future hardforks automatically with this setting. Then we don't need to manually take care of it, starting with Isthmus.

---

I'm going to apply this for Ash because its late in Japan right now